src/compile/header/index.ts

```diff
@@ -98,15 +97,15 @@ export function getHeaderGroups(model: Model, channel: HeaderChannel): VgMarkGro
 
 // 0, (0,90), 90, (90, 180), 180, (180, 270), 270, (270, 0)
 
-export function labelAlign(angle: number) {
+export function labelAnchor(angle: number) {
```

Comment:
Do we still have labelAngle property? 

If so, I wonder if replacing labelAlign with labelAnchor rule still make them nice?

Reply:
I'm playing with it and it looks like anchor does not do the same thing as align. Should I change it back?

Reply:
Maybe, let's look at it together at the scrum.